### PR TITLE
test: [M3-6608] - Add Linode power off, power on and reboot tests

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add regression tests for powering on/off and rebooting Linodes

## Major Changes 🔄
- Add regression tests to power on/off and reboot Linodes

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/linodes/switch-linode-state.spec.ts"
```

### CODE REVIEW

File: packages/manager/.changeset/pr-9980-added-1701986820847.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Added
```
Comment (carrillo-erik) [inline packages/manager/.changeset/pr-9980-added-1701986820847.md:2]: I had a similar PR where I only added new tests and was unsure of which identifier to use for the changeset. In my case I ended up going with `Tests`. My decision was based on the descriptions found under the Commit Types in the Pull Request Template:

`test`: New tests or changes to existing tests. Does not change the production code.

**Comment by carrillo-erik** [comment]
Looks good. I ran the tests locally and verified that they all passed ✅ ✅ ✅

Screenshot

**Review by carrillo-erik — APPROVED** [review]

File: packages/manager/.changeset/pr-9980-added-1701986820847.md (hunk 1 above)
Comment (coliu-akamai) [reply packages/manager/.changeset/pr-9980-added-1701986820847.md:2]: +1 to using tests for the changeset category

**Review by coliu-akamai — APPROVED** [review]
confirmed all specs passed - thanks Cassie!

File: packages/manager/cypress/e2e/core/linodes/switch-linode-state.spec.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            .should('be.visible')
+            .should('be.enabled')
+            .click();
+        });
+
+      cy.get(`[data-qa-linode="${linode.label}"]`)
+        .should('be.visible')
+        .within(() => {
+          containsVisible('Shutting Down');
+          cy.contains('Offline', { timeout: 300000 }).should('be.visible');
```
Comment (tyler-akamai) [inline packages/manager/cypress/e2e/core/linodes/switch-linode-state.spec.ts:45]: is 5 minutes too long? Not sure what our standards are.

**Comment by tyler-akamai** [comment]
LGTM, nice clean tests. I dont have baker access, but not sure whats causing the failure.
Screenshot 2023-12-17 at 3 12 36 PM

**Review by tyler-akamai — APPROVED** [review]

**Pull request merged**